static/styles/zulip.scss

```diff
@@ -1178,7 +1182,9 @@ td.pointer {
 }
 
 .message_failed {
+    left: 20px;
```

Comment:
Is it possible for you to not use fixed positioning for aligning the buttons? Would strongly prefer a flexbox solution, but if it isn't possible, let me know.